lib/hider/accounts/user.ex

```diff
@@ -73,4 +75,15 @@ defmodule Hider.Accounts.User do
       validate_required(changeset, :password)
     end
   end
+
+  def put_encrypt_cpf(changeset) do
+    cpf = get_field(changeset, :cpf)
+
+    cpf_encrypted = AES.encrypt(cpf)
+    cpf_blind_index = Argon2.hash(cpf)
```

Comment:
Generate the blind index, this is better on unique indexes, cause it reviews less meta data, (you can't match different rows with the same value.